### PR TITLE
[ci] Skip Android webview FTL tests on stable

### PR DESCRIPTION
**Pull request by stuartmorgan**
These tests crash on stable due to
https://github.com/flutter/flutter/issues/96661 now that 2.10 has been
released to stable.

Fixes out-of-band tree closure from the 2.10 release.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran [the auto-formatter]. (Unlike the flutter/flutter repo, the flutter/plugins repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [ ] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[the auto-formatter]: https://github.com/flutter/plugins/blob/master/script/tool/README.md#format-code
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

**Review by ditman — APPROVED** [review]

**Comment by ditman** [comment]
(This will need to land in red to fix the submit-queue check)

**Comment by ditman** [comment]
Still failing?

https://console.firebase.google.com/project/flutter-cirrus/testlab/histories/bh.1ddca77c39c73e39/matrices/8305578407344643451

File: .cirrus.yml [hunk 1]
```diff
@@ -221,7 +221,15 @@ task:
         # Native integration tests are handled by firebase-test-lab below, so
         # only run unit tests.
         # Must come after build-examples.
-        - ./script/tool_runner.sh native-test --android --no-integration --exclude script/configs/exclude_native_unit_android.yaml
+        # TODO(stuartmorgan): Remove this condition once
+        # https://github.com/flutter/flutter/issues/96661 is fixed and
+        # cherry picked. Currently webview_flutter tests crash on stable on
+        # Android so must be skipped.
+        - if [[ "$CHANNEL" == "stable" ]]; then
+        -   ./script/tool_runner.sh native-test --android --no-integration --exclude script/configs/exclude_native_unit_android.yaml,webview_flutter
```
Comment (ditman) [inline .cirrus.yml:229]: Does this `if` belong in line 241 of the file? `./script/tool_runner.sh firebase-test-lab` instead?

File: .cirrus.yml [hunk 2]
```diff
@@ -221,7 +221,15 @@ task:
...
+        - else
+        -   ./script/tool_runner.sh native-test --android --no-integration --exclude script/configs/exclude_native_unit_android.yaml
+        - fi
       firebase_test_lab_script:
         # Unsetting CIRRUS_CHANGE_MESSAGE and CIRRUS_COMMIT_MESSAGE as they
         # might include non-ASCII characters which makes Gradle crash.
```
Comment (ditman) [inline .cirrus.yml:235]: Argh, I cannot comment in unchanged code :P, but a few lines below this :P

File: .cirrus.yml (hunk 1 above)
Comment (stuartmorgan) [reply .cirrus.yml]: Doh!

**Comment by stuartmorgan** [comment]
🤦🏻 

Thanks. I hate changing CI scripts that can't be tested locally.

**Comment by ditman** [comment]
No problem! I hope this goes through now :D

**Comment by stuartmorgan** [comment]
Landing on red to fix tree.

**Pull request merged**